examples/AccountManagement/GetAccountHierarchy.cs

```diff
@@ -0,0 +1,218 @@
+    /// portions of the hierarchy for each accessible customer.
+    /// </summary>
+    public class GetAccountHierarchy : ExampleBase
+    {
+        // private const string MANAGER_CUSTOMER_ID = "3955750366";//"INSERT_MANAGER_CUSTOMER_ID_HERE";
+        private const int PAGE_SIZE = 1000;
+
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            GetAccountHierarchy codeExample = new GetAccountHierarchy();
+            Console.WriteLine(codeExample.Description);
+            long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+            
+            try
+            {
+                codeExample.Run(new GoogleAdsClient(), customerId);
+            }
+            catch (GoogleAdsException adsException)
+            {
+                Console.WriteLine("Failure:");
+                Console.WriteLine($"Message: {adsException.Message}");
+                Console.WriteLine($"Failure: {adsException.Failure}");
+                Console.WriteLine($"Request ID: {adsException.RequestId}");
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description =>
+            "This code example gets the account hierarchy of a specified manager account. " +
+            "If you don't specify a manager customer ID, the example will instead print the hierarchies " +
+            "of all accessible customer accounts for your authenticated Google account.";
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="googleAdsClient">The Google Ads client instance.</param>
+        /// <param name="customerId">The customer ID.</param>
+        public void Run(GoogleAdsClient googleAdsClient, long? customerId = null)
+        {
+             // googleAdsServiceClient = googleAdsClient.GetService(Services.V3.GoogleAdsService);
+            
+            // List of Customer IDs to handle.
+            ArrayList seedCustomerIds = new ArrayList();
+            
+            // If a Manager ID was provided above, it will be the only ID in the list.
+            // Otherwise, we will issue a request for all customers accessible by this authenticated Google account.
+            if(customerId.HasValue)
+            {
+                seedCustomerIds.Add(customerId);
+            }
+            else
+            {
+                Console.WriteLine("No manager customer ID is specified. The example will print the hierarchies of " + 
+                                  "all accessible customer IDs:");
+                CustomerServiceClient customerServiceClient = googleAdsClient.GetService(Services.V3.CustomerService);
```

Comment:
Maybe you can get this earlier in the method when you get the other service as well?

Reply:
Fixed.